### DISCUSSION TITLE
error warnings problem in vs code

### DISCUSSION THREAD

**Discussion by sagoto80** [Cyfrin/foundry-full-course-cu#1231] [Q&A]
If my internet connection is not available in the starting of running vscode,it does not show any error warning.
Additionally it does not work correctly.

**Comment by xyz899** [comment]
Hi,
extensions require internet access to update or fetch data, they might not work correctly under these conditions. The lack of error warnings might be because VSCode doesn't require internet access for basic local stuff so try this : 
- Restart VSCode
- Check your internet connection and test it 
- Check if there's a problem with the extensions you're using ( in that case Juan Blanco's or Hardhat's)
- Restart your Machine ( sometimes the files aren't getting loaded properly and it causes this

Ensuring a stable internet connection is necessary

**Comment by sagoto80** [comment]
thanks for your suggestion.
can you help me for this problem:https://github.com/Cyfrin/foundry-full-course-f23/discussions/1181
